[New Term] C++ Math Functions: `rint()` (#1071)
Codecademy/docs: content/cpp/concepts/math-functions/terms/rint/rint.md

---
Title: 'rint()'

> Description: 'The rint() function in C++ rounds the argument to an integral value using the current rounding mode.'

Reviewer: ```suggestion
Description: 'Rounds the argument to an integral value using the current rounding mode.'
```

Subjects:
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Arithmetic'
  - 'Methods'
CatalogContent:
  - 'learn-c-plus-plus'
  - 'paths/computer-science'
---

> 

Reviewer 2: ```suggestion

The **`rint()`** function rounds the argument to an integral value using the current rounding mode.

```

## Syntax

```pseudo
rint(x)
```

> Argument must be a `double`/`float`/`long double`, and the return value will be same type.

Reviewer 2: ```suggestion
The `x` parameter must be a `double`, `float`, or `long double`. The return value will be the same data type.
```

> 
> ## Return value
> 
> The rint() function rounds the argument x to an integral value, using the rounding direction specified by fegetround() and returns the value. By default, the rounding direction is set to 'to-nearest'. The rounding direction can be set to other values using fesetround() function.

Reviewer 2: ```suggestion

The rounding direction specified by the `fegetround()` function, with the default direction set as 'to-nearest'. The rounding direction can be set to other values using `fesetround()` function.
```

> The rounding direction specified by the `fegetround()` function, with the default direction set as 'to-nearest'. The rounding direction can be set to other values using `fesetround()` function.

Reviewer 2: ```suggestion
The rounding direction is specified by the `fegetround()` function, with the default direction set as 'to-nearest'. The rounding direction can be set to other values using `fesetround()` function.
```

## Example

> The following example uses the `rint()` function to find the value of x after rounding off:

Reviewer 2: ```suggestion
The following example uses the `rint()` function to find the value of `x` after rounding off:
```
